src/test/java/duke/EventTest.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+import duke.tasks.Event;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class EventTest {
+
+    @Test
+    void testToString() {
+        Event e = new Event("concert", LocalDate.parse("2021-01-22"));
+        assertEquals("[E][ ] concert (at: Jan 22 2021)", e.toString());
+    }
+
+    @Test
+    void taskMemo() {
```

Comment:
Maybe the function name could be something like this to be consistent :)
```suggestion
public void testTaskMemo() {
```